spec/chewy/rspec/mock_elasticsearch_response_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'spec_helper'
+
+describe :mock_elasticsearch_response do
+  before do
+    stub_model(:city)
+    stub_index(:cities) { index_scope City }
+    CitiesIndex.create
+  end
+
+  let(:dummy_query) { {} }
+
+  specify do
+    mock_elasticsearch_response(dummy_query)
+    CitiesIndex.client.search(dummy_query)
+  end
```

Comment:
This test is not testing anything, you should tests this with some `expect` against `ClientIndex.query(something).hits` using a similar idea on what was done with minitest `mock_elasticsearch_response` helper